modules/local/clipkit.nf

```diff
@@ -12,10 +12,10 @@ process CLIPKIT {
     )
 
     input:
-    tuple val(meta), path(fasta)   // Filepaths to the MSAs
+    path(fasta)   // Filepaths to the MSAs
```

Comment:
So you did end up just getting away from our usage of the tuple here - I think that probably makes good sense, since I don't believe we need to use any of the associated metadata here (or need to retain them in the output)